packages/Search/test/tstLinearBVH.cpp

```diff
@@ -60,13 +60,14 @@ TEUCHOS_UNIT_TEST_TEMPLATE_1_DECL( LinearBVH, tag_dispatching, NO )
     Kokkos::fence();
 
     DataTransferKit::BVH<NO> bvh( boxes );
-    Kokkos::View<int *, DeviceType> results;
+    int results[255];
+    unsigned int n_results;
     DataTransferKit::Point p1 = {0., 0., 0.};
-    DataTransferKit::BVHQuery<NO>::query( bvh, details::nearest( p1, 1 ),
-                                          results );
+    details::TreeTraversal<NO>::query( bvh, details::nearest( p1, 1 ), results,
+                                       n_results );
```

Comment:
How would this work? Your `results` live on the host, however, your `query` runs on the device. This results in invalid pointer, right?